src/Controller/AddTimeController.php

```diff
@@ -20,43 +21,29 @@ public function index()
         ]);
     }
 
-
     /**
      * @Route("/time-slot-submit", name="submit-add-slot", methods={"POST"})
```

Comment:
su route naming'u, tai matau nekazka :( as siunciau kazkada REST'o cheatsheet'a: https://devhints.io/rest-api cia labai sutrauktas info kaip turi atrodyti API'jus. Is principo, cia tu kuri sesija taip? Tai tavo route'as turetu buti toks: `POST /sessions`.
* `POST /sessions` apibrezia, kad tu kuri nauja sesija 
* `GET /sessions` - gauni sesiju sarasa 
* `GET /sessions/{id}` - gauni konkrecia sesija
* `PUT /sessions/{id}` - editini konkrecia sesija (nors kai kurie cia naudoja ir POST)
* `DELETE /sessions/{id}` - trini konkrecia sesija

Taip atrodo REST'inis API, nes time-to-submit yra labai netinkamas endoint'o namingas